backend/audit/cross_validation/check_ref_number_in_findings_text.py

```diff
@@ -26,6 +29,14 @@ def check_ref_number_in_findings_text(sac_dict, *_args, **_kwargs):
     in_use_references = set()
     errors = []
 
+    skip_validation_function = InvalidRecord.fields["validations_to_skip"]
+    if (
+        data_source == settings.CENSUS_DATA_SOURCE
+        and "check_ref_number_in_findings_text" in skip_validation_function
+    ):
+        # Skip this validation if it is a historical audit report with non-matching reference numbers
+        return errors

```

Comment:
This was approach was changed by Hassan, we can now just add the method name to [this](https://github.com/GSA-TTS/FAC/blob/main/backend/audit/intakelib/checks/runners.py#L151) dict

Reply:
This is a little different, here it is cross_validation (not intake validation)